File: ios/ToDo/Item.swift
//
//  Item.swift
//  ToDo
//
//  Created by Maximilian Alexander on 1/16/18.
//  Copyright © 2018 Maximilian Alexander. All rights reserved.
//

import RealmSwift

class Item: Object {
    
    @objc dynamic var itemId: String = UUID().uuidString
    @objc dynamic var body: String = ""
    @objc dynamic var isDone: Bool = false
    @objc dynamic var timestamp: Date = Date()
    
}

File: xamarin/XamarinToDoApp/XamarinToDoApp/Item.cs
using System;
using Realms;

namespace XamarinToDoApp
{
    public class Item: RealmObject
    {
        [PrimaryKey]
        public string itemId { get; set; } //set random uuid
        public string body { get; set; }
        public bool isDone { get; set; }
        public DateTimeOffset timestamp { get; set; }
    }
}
